libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -430,6 +474,47 @@ bool AP_Arming::battery_checks(bool report)
                 }
                 return false;
             }
+
+            float temperature = 0.0f;
+            // temperature minimum check
+            if (_min_temperature[i] > 0.0f) {
+                if (!_battery.get_temperature(temperature, i)) {
+                    if (report) {
+                        gcs().send_text(MAV_SEVERITY_CRITICAL, "PreArm: Battery %d not get the temp",
+                                i+1);
+                    }
+                    return false;
+                }
+                if (temperature < _min_temperature[i]) {
+                    if (report) {
+                        gcs().send_text(MAV_SEVERITY_CRITICAL, "PreArm: Battery %d temp %.1f below minimum %.1f",
```

Comment:
Too long; maybe say `min` rather than `minimum`

Reply:
I change "min" to "minimum", "max" to "maximum".